pkg/cmd/factory/factory.go

```diff
@@ -125,3 +150,59 @@ func (c *ClientFlags) GetCredential() *security.Credential {
 		CertAllowedCN: certAllowedCN,
 	}
 }
+
+func (c *ClientFlags) CompleteAuthParameters(cmd *cobra.Command) (err error) {
+	defer func() {
+		if err == nil {
+			if c.User == "" && c.Password != "" {
+				err = errors.ErrCredentialNotFound.GenWithStackByArgs("invalid atuhentication: password is specified without user")
+			}
+		}
+	}()
+	// If user is specified via command line, password should be specified as well.
+	if c.User != "" {
+		if c.Password == "" {
```

Comment:
empty password is a normal case

Reply:
When password is empty, we will ask the user to enter the password from the terminal to confirm that the password is deliberately set to empty.